src/main/java/frc/robot/Constants.java

```diff
@@ -1,70 +1,62 @@
+import static edu.wpi.first.units.Units.Kilograms;
 import static edu.wpi.first.units.Units.Meters;
 import static edu.wpi.first.units.Units.Pounds;
 import static edu.wpi.first.units.Units.RotationsPerSecondPerSecond;
 import static edu.wpi.first.units.Units.Second;
 
+import com.ctre.phoenix6.signals.GravityTypeValue;
 import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.system.plant.DCMotor;
 import edu.wpi.first.units.AngularAccelerationUnit;
+import edu.wpi.first.units.measure.Angle;
 import edu.wpi.first.units.measure.AngularAcceleration;
 import edu.wpi.first.units.measure.AngularVelocity;
 import edu.wpi.first.units.measure.Distance;
 import edu.wpi.first.units.measure.Mass;
+import edu.wpi.first.units.measure.MomentOfInertia;
 import edu.wpi.first.units.measure.Velocity;
 import edu.wpi.first.wpilibj.RobotBase;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
 import frc.robot.subsystems.Elevator;
 
-public final class Constants {
-
-  /**
-   * This class defines the runtime mode used by AdvantageKit. The mode is always "real" when
-   * running on a roboRIO. Change the value of "simMode" to switch between "sim" (physics sim) and
-   * "replay" (log replay from a file).
-   */
-  public static final Mode simMode = Mode.SIM;
-
-  public static final Mode currentMode = RobotBase.isReal() ? Mode.REAL : simMode;
-
-  public static enum Mode {
-    /** Running on a real robot. */
-    REAL,
-
-    /** Running a physics simulator. */
-    SIM,
-
-    /** Replaying from a log file. */
-    REPLAY
-  }
-
-  public static final String CANivoreName = "Default Name";
-  public static final String RIOName = "rio";
+// Copyright 2021-2024 FRC 6328
+// http://github.com/Mechanical-Advantage
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// version 3 as published by the Free Software Foundation or
+// available in the root directory of this project.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
 
-  public static class MotorIDConstants {
+public final class Constants {
+  public static class CANIDs {
     // Rev Can Bus
+    // 1-8 Motor ID is reserved by the drivebase
     public static final int kScorerMotor = 9;
     public static final int kElevatorMotor1 = 10;
     public static final int kElevatorMotor2 = 11;
     public static final int kIntakeMotor = 12;
-    // CANivore Can Bus
+    public static final int kPivotMotor = 15; // FIXME: Change to correct ID
+    public static final int kConveyorMotor = 17; // FIXME: Change to correct ID
+    public static final int kConveyorSensor = 18; // FIXME: Change to correct ID -> 1
```

Comment:
The motor IDs of 13, 14, and 15 should be unused. I would use these just to make everything in order. I don't think the motors have been set to that yet, but it's an easy change.